CHANGES.rst

```diff
@@ -14,6 +14,9 @@ New features:
 
 Bug fixes:
 
+- Fix #46, when ``tiff_utils.process_tiff`` failed and a ``width referenced before assignment`` error occured.
```

Comment:
a bit confusing, when the changelog only speaks about tiff image processing and also jpeg and png handling is touched.